cadquery/cq.py

```diff
@@ -1722,6 +1744,37 @@ def radiusArc(self, endPoint, radius, forConstruction=False):
         else:
             return self.sagittaArc(endPoint, -sag, forConstruction)
 
+    def tangentArcEndpoint(self, endpoint, forConstruction=False, relative=True):
+        """
+        Draw an arc as a tangent from the end of the current edge to endpoint.
+
+        :param endpoint: point for the arc to end at
+        :type endpoint: 2-tuple or Vector
```

Comment:
Will it not work with 3-tuple as well? Maybe not the most common use case but I can think of some applications.

Reply:
It should work just fine in it's current form with 3-tuples, creating 3D arcs. I just didn't write any tests for it so I didn't want to advertise that in the docstring.

Reply:
Docstring now mentions 3-tuples.